src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +13,17 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] resultString = text.replaceAll("[^0-9-]", ":").split(":");
+        ArrayList<Integer> numberList = new ArrayList<>();
+        for (int i = 0; i < resultString.length; i++) {
+            if (resultString[i].length() > 0) {
+                numberList.add(Integer.parseInt(resultString[i]) * 2);
+            }
+        }
+        int[] resultArray = new int[numberList.size()];
+        for (int i = 0; i < resultArray.length; i++) {
+            resultArray[i] = numberList.get(i);
+        }
+        return resultArray;
```

Comment:
try to look if have method  to convert list to array

Reply:
There is toArray() method, but it returns Object[]... However, I've found another way to split the String at the very beginnig, so that I get int[] with less lines of code.